kin-sdk/src/main/java/kin/sdk/AccountInfoRetriever.java

```diff
@@ -41,9 +37,8 @@ Balance getBalance(@NonNull String accountId) throws OperationFailedException {
                 throw new OperationFailedException("can't retrieve data for account " + accountId);
             }
             for (AccountResponse.Balance assetBalance : accountResponse.getBalances()) {
-                if (kinAsset.isKinAsset(assetBalance.getAsset())) {
+                if (assetBalance.getAsset().getType().equalsIgnoreCase("native")) {
```

Comment:
can't we compare to the class of `AssetTypeNative` instead of string comparison?